autocrop/cli.py

```diff
@@ -188,6 +199,14 @@ def confirmation(question):
         notification_str = "Please respond with 'y' or 'n'"
         print(notification_str)
 
+def chk_extension(p):
+    """Check if the extension passed is valid or not."""
+    inp_extension = str(p)
+    inp_extension = inp_extension.lower()
+    if inp_extension in COMBINED_FILETYPES:
+        return p
+    elif "."+inp_extension in COMBINED_FILETYPES:
+        return p
```

Comment:
What happens if the extension isn't in the list, though?

A type parsing function for `argparse` will need to raise an error if it's problematic, i.e. see the `size` function.

Reply:
Done dev......